orx-jvm/orx-midi/src/main/kotlin/MidiTransceiver.kt

```diff
@@ -294,4 +295,39 @@ fun Program.openMidiDevice(name: String): MidiTransceiver {
     val actualName = matchingDevice?.name ?: name
 
     return MidiTransceiver.fromDeviceVendor(this, actualName)
```

Comment:
Can openMidiDevice be implemented using `return openMidiDevilceOrNull(name) ?: error("no MIDI device found for query  '$name'")` . This has the benefit of clearer error messages.

Reply:
How does https://github.com/openrndr/orx/pull/312/commits/6d2a4c5e0b15b20f7455ea027116423fa88e7e42 look like?

Should all `throw`s be `error`? Or leave them as they were?